lib/everypolitician.rb

```diff
@@ -105,7 +108,40 @@ def popolo_url
     end
 
     def legislative_periods
-      @legislative_periods ||= raw_data[:legislative_periods]
+      @legislative_periods ||= raw_data[:legislative_periods].map do |lp|
+        LegislativePeriod.new(lp, self, self.country)
+      end
+    end
+  end
+
+  class LegislativePeriod
+    attr_reader :id
+    attr_reader :name
+    attr_reader :slug
+    attr_reader :legislature
+    attr_reader :country
+    attr_reader :raw_data
+
+    def initialize(legislative_period_data, legislature, country)
+      @id = legislative_period_data[:id]
+      @name = legislative_period_data[:name]
+      @slug = legislative_period_data[:slug]
+      @legislature = legislature
+      @country = country
+      @raw_data = legislative_period_data
+    end
+
+    def start_date
```

Comment:
Just wasn't something I needed for GB, but I might as well add it now to avoid confusion in the future.